### ISSUE TITLE
Move location of search main

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#115]
Right now we have:

```
$ python -m pyserini.search.pysearch
usage: pysearch.py [-h] -index path -topics path -output path
```

Maybe let's move to:

```
$ python -m pyserini.search
usage: pysearch.py [-h] -index path -topics path -output path
```

What do you think @yuki617 ?

**Comment by yuki617** [comment]
Do you mean to put the main function into __init__.py? I feel that can cause a little confusing, since pyserini.search is a package instead of a file.

**Comment by lintool** [comment]
Yea, I think moving into init should make it work? Try it out?

**Comment by yuki617** [comment]
Yea, I have tried to add it in the __init__.py before. But it shows an
error message saying pyserini.search is a package and cannot be directly
executed.

**Comment by lintool** [comment]
Apparently, I have much more to learn in the ways of Python:

https://stackoverflow.com/questions/35258923/python-main-and-init-proper-usage/35258967
https://docs.python.org/3/library/__main__.html

**Comment by lintool** [comment]
Closed by #117
